docs/usage/vscode.mdx

```diff
@@ -15,6 +15,9 @@ In the latest [PowerShell](https://marketplace.visualstudio.com/items?itemName=m
 
 Actually there is a bug, and the option is called Enable Legacy Code Lens, and is enabled by default and should be disabled for Pester 5. 😁 Take advantage of this and go try it right now!
 
+**Instructions (for VSCode on Windows)** In Visual Studio Code, select **File | Preferences | Settings** (or type **Ctrl+,**). In the **Settings** window, under the **User** tab, select **Extensions | PowerShell Configuration**. Uncheck the **Pester: Use Legacy Code Lens** checkbox. Close the **Settings** window.
```

Comment:
Generalize to open Settings -> ... to avoid being Windows/only?

Reply:
Sure, but I only have a Windows OS, so I don't know what the experience is elsewhere. How to proceed?

Reply:
AFAIK, VSCode is mostly identical cross-platform besides maybe the menu bar (+ ctrl/cmd) on Mac. So I'd just skip the first sentence and start directly with "In the Settings window...".